src/index.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
       rules: {
         'jest/no-alias-methods': 'warn',
         'jest/no-disabled-tests': 'warn',
+        'jest/no-commented-out-tests': 'warn',
```

Comment:
changing the recommended config is a breaking change.

Not sure this needs to be in the recommended config, at all, but we can revisit that before the next release 🙂

Reply:
Ah, I didn't realise this will modify recommended config. On the other hand, I think it's ok to warn since warnings usually don't fail with tools like lint-staged, etc.

Reply:
People can run eslint with a flag failing on warnings

Reply:
removed it